### ISSUE TITLE
Consider tx size

### ISSUE THREAD

**Issue by wjmelements** [ethgasstation/ethgasstation-adaptive-oracle#6]
Some transactions (for example https://etherscan.io/tx/0xec99eaff12b7100ac5428aa0737a12750362236da0ebd535b3f37077f8a743d0 ) consume large proportions of the block gas limit. These transactions will take longer to confirm than smaller transactions because they have to outbid larger portions of the pending tx pool. The current model does not take tx size into account. In particular, the Gas Used cell on the main page should affect confirmation time in addition to tx fee.

**Comment by bamos01** [comment]
Hi good point - you are correct but the model does take into account 'gas offered' by the transaction but not 'gas used' (since the latter requires a lot more overhead to get from the node and it is 'gas offered' that the miners see).  In my models, there is not a linear relationship between 'gas offered' and confirmation time  - so it dichotomized based on whether gas offered is > 15% of the gas limit- this is used in the confirmation time estimates.

**Comment by wjmelements** [comment]
My main objection is that increasing the gas used on the main page's "Gas-Time-Price Estimator" does not affect confirmation time. I haven't noticed any dichotomy.

I don't expect the model to be linear because the distribution of gas prices in a block is non-linear. Probably there aren't many large transactions in the dataset either.

Considering gas used instead of gas offered would be more accurate, since gas used is what counts toward block gas limit. Gas offered can be anywhere between gas used and the block gas limit, and several services provide substantially more gas than required.

**Comment by bamos01** [comment]
Yes- but remember - the miners don't know the gasused until after they mine the transaction- when they are seeing transactions in the txpool all they know is the gas offered.  Your right - i haven't integrated that gas offered into that calculator - it doesn't cover special use cases like high gas transactions.
